packages/ui/package.json

```diff
@@ -36,9 +36,12 @@
     "@radix-ui/react-dropdown-menu": "^2.0.4",
     "@radix-ui/react-scroll-area": "^1.0.3",
     "@radix-ui/react-separator": "^1.0.2",
+    "@radix-ui/react-slot": "^1.0.1",
     "@radix-ui/react-switch": "^1.0.2",
     "@radix-ui/react-tabs": "^1.0.3",
+    "@radix-ui/react-toggle": "^1.0.2",
     "@radix-ui/react-toggle-group": "^1.0.3",
+    "@radix-ui/react-use-controllable-state": "^1.0.0",
```

Comment:
I'm having trouble finding any documentation on this one.  Is this OK to use?

Reply:
Probably not. Though, it is exactly what I was looking for here. I could re-write it and link to original implementation as a reference.

Reply:
Found this discussion specifically about this and think it may be more stable to re-implement this ourselves.

https://github.com/radix-ui/primitives/discussions/1146

Reply:
Yeah, good find.  We have separate packages now so we've got a good home for it.  :smile:

Reply:
Okay, I wasn't sure if they should be their own package or not since they are only being used in the one place 🤔 

But, I added them in [re-implement radix hooks](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6077/commits/8a0f4c9f0b4d8c2307a9259908558dce13f6814e). Let me know if we should make them separate packages or not or If you believe they should be placed elsewhere.

Reply:
Oh yeah, I like this in the hooks package!